src/main/java/me/totalfreedom/totalfreedommod/command/Command_cage.java

```diff
@@ -16,7 +16,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.ADMIN, source = SourceType.BOTH)
-@CommandParameters(description = "Place a cage around someone with certain blocks, or someone's player head.", usage = "/<command> <purge | <partialname> [head | block] [playername | blockname]")
+@CommandParameters(description = "Place a cage around someone with certain blocks, or someone's player head.", usage = "/<command> <purge | <player> [head | block] [playername | blockname]")
```

Comment:
partialname is actually supported whenever the syntax calls for player, as whenever bukkit does its whole thing with getPlayer it is a partialname thing, so i changed it to just be player because thats what the majority of the rest of the code used

Reply:
That's fair, I know some commands historically required the full correct player name for it to work, but if that's not the case and player will work with partial players as well that's fine.

Reply:
I think there might still be a few cases where the direct playername is used but they are few and far between